### ISSUE TITLE
Screen reader: Apply button not announced

### ISSUE THREAD

**Issue by 1j01** [1j01/midi-recorder#9]
With ChromeVox at least, the Apply button is not announced when you activate the Learn Range feature, because it uses the same button and just changes the text label.

I should either make it a new button, or perhaps unfocusing and refocusing it would work fine.
